easy_phi/app.py

```diff
@@ -296,7 +297,12 @@ def get(self):
 
         self.set_header('Content-type', 'application/javascript')
 
-        temp_id = int(time.time()*1000)
+        temp_id = int(time.time()*100000)
+        # time.time() has precision of 1/100000 of a second. Yet it is not a
```

Comment:
fix for issue from release notes in 0.2.6 (widgets containers sometime get into wrong place)